### ISSUE TITLE
Error when encoding RS256

### ISSUE THREAD

**Issue by julman99** [vandium-io/jwtgen#4]
When using RS256, there is an error when the private key is trying to be loaded from disk. This happens both in Mac and Linux.

```
/usr/local/lib/node_modules/jwtgen/bin/jwtgen.js:168
    builder.privateKeyFromPath( argv.p );
```

**Comment by virusdave** [comment]
Any progress on this?  The published npm package still fails

**Comment by dc00p** [comment]
Fixed in v2.2.0
